src/main/java/core/basesyntax/Drawing.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+//interface for displaying each figure's info
+public interface Drawing {
+    String toDrawInfo();
```

Comment:
```suggestion
    String drawInfo / displayInfo();
```
назви методів зазвичай дієслова